jsonschema-validation.xml

```diff
@@ -1043,8 +1043,14 @@
                     is going to be removed in the future.
                 </t>
                 <t>
-                    A root schema containing "deprecated" with a value of true indicates the entire
-                    root schema MAY be removed in the future.
+                    A root schema containing "deprecated" with a value of true indicates that
+                    the entire resource being described MAY be removed in the future.
+                </t>
+                <t>
+                    When the "deprecated" keyword is applicable to an array, it's meaning will
+                    vary depending on how "items" is used. If "items" is a single schema, it will
```

Comment:
> The previous sentance is saying:
> 
> "items": {
>    "type": "string",
>    "deprecated": true
> }
> 
> based on your understanding you think im suggesting it jumps up a level somehow?

`When the "deprecated" keyword is applicable to an array...`
This means, when the keyword is in a schema which is applied to an array.
`maxItems` would be an example of a keyword which is applicable to an array.
The value of `items` is an array of subschemas, which are then applied to each item in the array.

Based on your previous comments, what you mean is "When the `deprecated` keyword is applied to an item in an array by means of `items`, if `items` is a single schema, the deprecation relates to the whole array, while if `items` is an array of schemas, the deprecation relates to the corrosponding item according to the subschemas position.

Reply:
Ok I've put in that exact wording! Thumbs? :D